packages/core/src/contrib/Queries.ts

```diff
@@ -281,6 +282,52 @@ export const rectLineDist = (
   ]);
 };
 
+export const polygonSignedDistance = (
```

Comment:
Is this `polygonSignedDistance` function used anywhere? Notice that `Functions.ts` already has `signedDistancePolygon` which apparently does the same thing?

Reply:
No, good catch.  It seems this was just kept by the automatic merge (Git did not detect that it was renamed by the abstract shape PR).